enterprise/server/vmexec/vmexec.go

```diff
@@ -76,6 +76,20 @@ func (x *execServer) Initialize(ctx context.Context, req *vmxpb.InitializeReques
 	return &vmxpb.InitializeResponse{}, nil
 }
 
+func (x *execServer) UnmountWorkspace(ctx context.Context, req *vmxpb.UnmountWorkspaceRequest) (*vmxpb.UnmountWorkspaceResponse, error) {
```

Comment:
nit: would be nice to send the vmexec changes in a separate cl

Reply:
Done: https://github.com/buildbuddy-io/buildbuddy/pull/1674